basic_pipeline/1_Introduction.md

```diff
@@ -0,0 +1,55 @@
+# Introduction
+In this tutorial, we will create our very own Membrane Framework [pipeline]() consisting of our custom [elements]() which will fulfill the multimedia processing task.
+Despite the fact that the multimedia processing task we will be facing will be really simple, we will need to deal with some problems occurring in real-life scenarios.
+At the same time, we will make ourselves comfortable with some concepts of multimedia streaming as well as get in touch with the nomenclature used in this field. 
+Prepare for an adventure! 
+# Environment preparation
+In order to be able to proceed with the tutorial, you need to have Elixir installed: [How to install Elixir](https://elixir-lang.org/install.html).
+
+Once you are ready with the Elixir, you can get the project template we have prepared for you:
+```
+git clone https:/github.com/membraneframework/membrane_getting_started_tutorial
+cd membrane_getting_started_tutorial
+git checkout template/start
+```
+As you can see, the template's code is put on the `template/start` branch of the repository.
+In the repository, there is also a `template/end` branch, where you can find the completed template.
+If you find yourself lost during the tutorial feel free to check the implementation proposed by us, put on this branch.
+
+# Use Case
+Imagine that there is a conversation occurring between two peers and the chat is based on a question-answer scheme.
+Each of the peers sends its part of the conversation (that means - the first peer sends questions and the second peers sends the answers to them). We refer to each sentence sent by the peers as a **frame**.
+However, due to network limitations each frame sent by the peers is fragmented since only a few chars can be sent at the same time - that means, a single frame is sent as a sequence of **packets**.
```

Comment:
Add a note that it is called MTU and it is usually 1500B, but we lowered 1500 to a few to make the example easier to visualize.

Reply:
done